mimic/test/test_util.py

```diff
@@ -83,3 +83,9 @@ def test_unicode_body(self):
         self.failureResultOf(
             request(self, Resource(), b"POST", b"", u"not bytes")
         )
+
+    def test_bad_request(self):
```

Comment:
No documentation reference exists, because the error format is completely undocumented in the Nova docs.  All we know is that a 400 result happens.  Thus, Mimic is mirroring Rackspace's implementation of Nova, but it's not guaranteed to match, e.g., Nebula's implementation.